### ISSUE TITLE
Touch coordinates are offset in Genymotion

### ISSUE THREAD

**Issue by ashes999** [Garoe/gosu-android#19]
Hi,

I seem to have encountered a strange bug. When printing out the coordinates on `touch_moved`, it seems like (on Genymotion, at least) the coordinates are offset by some pre-determined amount.

When I click/move in an area around the top-left corner of the screen, I get `x=0, y = 56.476` (approximately), consistently.

It could be an emulator-specific bug. I don't have a device handy to test on. I know Bluestacks had its own strange bugs in the past.

**Comment by Gara-Dorta** [comment]
Indeed, on my physical device they are offset as well. I suspect they changed the default behaviour at some point. One easy solution would be to add an opposite  offset to the touch, to compensate for the window position.
The touch event is received  [here](https://github.com/Garoe/gosu-android/blob/26245a42756bb3e043f5a4385e4e546b6aae5edb/lib/gosu_android/main-window.rb#L43) and the class that is used is a [GLSurfaceView](http://developer.android.com/reference/android/opengl/GLSurfaceView.html). Ideally it'll be easier than that, and the GLSurfaceView would have some option to set the coordinate system as absolute or relative to the window position, I haven't actually check. Do you think you that is enough information to fix it?

**Comment by ashes999** [comment]
Yes, I think that's enough information. Unfortunately, I have to move on to other projects, and I don't think I will be spending time on this right now. Hopefully, I will revisit these bugs later.
